docker/README.md

```diff
@@ -0,0 +1,87 @@
+# Docker's version of `amc2moodle`
+
+A docker version of `amc2moodle` is available and it starts a daemon server to automatically execute `amc2moodle` or `moodle2amc`.
+
+Alternatively, it is also possible to run an interactive shell in the docker to recover standard `amc2moodle` command line interface.
+
+# Installation
+
+The container can be downloaded using the following command 
+```
+docker pull ghcr.io/luclaurent/amc2moodle:latest
+```
+
+The container proposes to mount two volumes:
+    - `/tmp/work` (**mandatory mount**) that must be link to a specific folder on the host computer
+    - `/tmp/daemon` (optional mount) that store the log file of the daemon
+
+For instance, the container can be run in CLI (in detached mode) with the following command:
+```
+docker run --name amc2moodle -d -v "DIRA:/tmp/work" -v "DIRB:/tmp/daemon" ghcr.io/luclaurent/amc2moodle
```

Comment:
same luclaurent -> nennigb for consistency

Reply:
Done!